agora-sample/Sample/Shared.hs

```diff
@@ -190,21 +197,23 @@ authorityToken = authorityTokenFromGovernor governor
 authorityTokenSymbol :: CurrencySymbol
 authorityTokenSymbol = authorityTokenSymbolFromGovernor governor
 
-proposalTimingConfig :: ProposalTimingConfig
-proposalTimingConfig =
+defaultProposalTimingConfig :: ProposalTimingConfig
+defaultProposalTimingConfig =
   ProposalTimingConfig
     { draftTime = 50
     , votingTime = 1000
     , lockingTime = 2000
     , executingTime = 3000
     }
```

Comment:
Use [`data-default-class`](https://hackage.haskell.org/package/data-default-class-0.1.2.0/docs/Data-Default-Class.html#t:Default) for this.